pkg/shared/families/malware/yara/yara.go

```diff
@@ -112,6 +128,27 @@ func (s *Scanner) Run(sourceType utils.SourceType, userInput string) error {
 			return
 		}
 
+		// If the (stdout lines)/(stderr + stdout lines) is less than `errThreshold` the error threshold will be reached.
+		if util.IsErrorThresholdReached(scanErrCount, outputLines+scanErrCount) {
+			s.sendResults(retResults, fmt.Errorf(
+				"scanner error threshold (%.2f%%) is reached, nuber of errors=%d, sample of errors: %w",
+				util.ErrThreshold*100, // nolint: gomnd
+				scanErrCount,
+				errors.Join(scanErrSamples...),
+			))
+			return
+		}
+		// If the (successfully parsed lines)/(stdout lines) is less than `errThreshold` the error threshold will be reached.
```

Comment:
"If the parse failures / lines parsed is greater than the err threshold then ..."

Reply:
https://github.com/openclarity/vmclarity/pull/732/commits/7d454c033f9373ffe7bed7259fb80514240d9037